src/types.ts

```diff
@@ -1,3 +1,5 @@
+import * as DOMPurify from 'dompurify';
```

Comment:
What about putting all these types in a `types.ts` file in `sanitizer` folder? I think it would help keep things separated